spec/support/shared_examples/export_to_spreadsheet.rb

```diff
@@ -59,9 +58,9 @@
     end
 
     it 'adds all the data to the spreadsheet' do
-      data = (existing_data + new_data).map(&:to_row)
+      data = (existing_data + new_data).map(&subject.method(:present))
```

Comment:
I'm thinking it would also be nice to use a similar "expected_spreadsheet_rows" as above.

I find having the expected row as part of the spec much more readable.

Reply:
Or maybe, the spec names aren't clear as to what functionality they care about.  It's quite possible that this spec shouldn't be concerned about the specific presentation of the data, in which case can we test what it does care about without needing to be affected by potential presentational changes in the future.

Reply:
Changed the test setup so that hopefully it's a lot clearer:

The initial data variable was the sum of `existing data` and `new data` being transformed to be appended to a blank worksheet. Renamed the data variable to be `full_worksheet_data` and have also used the `expected_spreadsheet_rows` variable (renamed: `expected_new_spreadsheet_rows`) in the setup.